### PR TITLE
Decouple release notes from doc releases

### PR DESCRIPTION
**Pull request by kprosise**
Replaced release note pages with external links.
These point to Markdown documents on main, allowing release notes to easily be updated post-release.

This is a follow up to PR #718.

QA Steps: Checked rendered output and tested links

This commit addresses FFTK-3256

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

This commit is a quick-fix for decoupling the release notes from the docs, allowing the notes to easily be updated post release. This will likely be replaced with a Sphinx focused solution in the future.

## Checklist

* [x] View HTML in a browser to check rendering.
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for dd048be5fcb2c8813ca855d03bc2049cab232555 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2677/docs/artifacts/html/index.html

File: source/index.rst [hunk 1]
```diff
@@ -97,11 +97,12 @@ OE/Yocto Project, the Linux microPlatform™ and Docker®.
...
 
 .. toctree::
-   :maxdepth: 1
+   :maxdepth: 2
    :caption: Release Notes
    :name: sec-release-notes
 
-   release-notes/release-notes
+   v92 <https://github.com/foundriesio/docs/blob/main/release-notes/rn_v92.md>
+   v93 <https://github.com/foundriesio/docs/blob/main/release-notes/rn_v93.md>
```
Comment (vanmaegima) [inline source/index.rst:105]: Let's do latest first
```suggestion
   v93 <https://github.com/foundriesio/docs/blob/main/release-notes/rn_v93.md>
   v92 <https://github.com/foundriesio/docs/blob/main/release-notes/rn_v92.md>
```

Comment (kprosise) [reply source/index.rst:105]: Thanks for catching that! I had that in mind, but seemed to slip!

**Comment by doanac** [comment]
Docs for 39137d1193b3d21478ca4c109d9a894dff78a704 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2678/docs/artifacts/html/index.html

**Review by angolini — APPROVED** [review]

**Comment by doanac** [comment]
Docs for eb13013f50f263ec1a7ef9edc32b5c1ab7581872 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2679/docs/artifacts/html/index.html

**Comment by angolini** [comment]
I only want to note that it looks weird to have a link in the HTML that points to the GitHub source code.

For me, it's not a "decouple". And for me, it would be better to work for dot releases instead of adding a hack here.

**Comment by kprosise** [comment]
> I only want to note that it looks weird to have a link in the HTML that points to the GitHub source code.
> 
> For me, it's not a "decouple". And for me, it would be better to work for dot releases instead of adding a hack here.

This is definitely not a long term solution, changing doc release process to include dot releases is definitely something we should consider. My goal here was to having something implemented in time for v94.

**Pull request merged**